quality/errorprone/android.gradle

```diff
@@ -1,10 +1,16 @@
 apply plugin: 'net.ltgt.errorprone'
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.4.6"
-  testAnnotationProcessor "com.uber.nullaway:nullaway:0.4.6"
+  if(JavaVersion.current().getMajorVersion().toInteger() < 9) {
+    // see https://github.com/tbroyer/gradle-errorprone-plugin#jdk-8-support
+    errorproneJavac("com.google.errorprone:javac:9+181-r4173-1")
+  }
+  errorprone("com.google.errorprone:error_prone_core:2.3.2")
+  errorprone("com.uber.nullaway:nullaway:0.4.6")
+  annotationProcessor "com.uber.nullaway:nullaway:0.6.0"
 }
 
 tasks.withType(JavaCompile) {
-  options.compilerArgs += ["-Xep:NullAway:ERROR", "-XepOpt:NullAway:AnnotatedPackages=com.uber"]
+  options.errorprone.errorproneArgs << "-XepDisableWarningsInGeneratedCode"
+  options.errorprone.errorproneArgs << "-Xep:NullAway:ERROR" << "-XepOpt:NullAway:AnnotatedPackages=com.uber"
```

Comment:
If you don't pull the `com.google.errorprone:javac:9+181-r4173-1` compiler you can keep using `options.compilerArgs`.

Reply:
but JDK8 needs to pull that, so for sake of backward compatibility i think this one is better.
or do you see any other drawbacks of using `options.errorprone.errorproneArgs` ?

Reply:
None, but I didn't need that 🤔

Reply:
yeah that kinda bugs me too... why did it fail on my jdk8 then? 💁‍♀️

Reply:
Ahh right, never mind. I stayed on an old plugin version and only updated error prone itself so that I didn't need all this, but version `0.6` of the plugin definitely need it! 👍